includes/class-wc-gateway-amazon-payments-advanced.php

```diff
@@ -2930,4 +2942,23 @@ public function update_address_details_for_classic( $payload, $checkout_session_
 
 		return $payload;
 	}
+
+	/**
+	 * Get the estimated order amount from the cart totals.
+	 *
+	 * @return array
+	 */
+	private static function get_estimated_order_amount() {
+
+		if ( is_null( WC()->cart ) ) {
+			return array();
+		}
+
+		return wp_json_encode(
```

Comment:
`wp_json_encode` returns `string|false`, but the method doc-block states it should be an array. What's intended to return here?

Reply:
Good catch, code was working by expecting a string to be returned. Updated the doc comment and the return when WC()->cart was null

Reply:
THanks!